{"uuid": "6ede24d1-f244-4dd5-b7d1-02e167f65b97", "children": ["4c45be31-8715-46b1-b59d-a7772710bdb9"], "befores": [{"name": "classSetup", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'driver'\n", "trace": "  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 887, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"F:\\AGEST\\Selenium\\SeleniumPython\\tests\\test_login.py\", line 16, in classSetup\n    self.login_page = LoginPage(self.driver)\n                                ^^^^^^^^^^^\n"}, "start": 1721026728013, "stop": 1721026728013}], "start": 1721026728013, "stop": 1721026728013}